L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringHistoImpl.cc

```diff
@@ -326,40 +458,69 @@ std::vector<GlobalPoint> HGCalMulticlusteringHistoImpl::computeThresholdSeeds( c
 
 
 std::vector<l1t::HGCalMulticluster> HGCalMulticlusteringHistoImpl::clusterSeedMulticluster(const std::vector<edm::Ptr<l1t::HGCalCluster>> & clustersPtrs,
```

Comment:
Is the old behaviour of this function still available? Users should be able to run with the same behaviour as before (except if there was a bug)

Reply:
I have re-included the old function as an overload - maybe there is a better option?

Reply:
Just to be sure I understand what's in the new method: is the old behaviour implemented in the new method and accessible for users with a given configuration?

Reply:
Yes the default clustering strategy is set to be "Fixed" in hgcalBackEndLayer2Producer_cfi.py which takes the usual dr_ parameter

Reply:
Ah ok. So in that case it is not needed to keep the old implementation since it is available with the new one. I just wanted to make sure that the old functionalities are still available.

Reply:
Right ok, that makes sense, I will remove the old function